README.md

```diff
@@ -177,11 +177,26 @@ To run the unitests run the following command in the terminal (while the virtual
 python -m unittest discover tests
 ```
 
+### Auto-formatting with black
+
+We use [_Black_](https://github.com/psf/black) to format code automatically so that we don't have to worry about clean and
+readable code. To install _Black_:
+
+```
+pip install black
+```
+
+To run black:
+
+```
+black .
```

Comment:
```suggestion
black -S .
```

Reply:
Black converts single quotes to double quotes which I don't think is good. We should change it. By adding -S flag, it will not convert single quotes to double

Reply:
Interesting!

I myself like Black's methodology of "double quotes everywhere" so that's it's consistent, but I understand it might not be the same for everyone else.

@isabelcosta Thoughts?

Reply:
I like that yes :)